### PR TITLE
[Ryujinx.Tests.Memory] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.Tests.Memory/MultiRegionTrackingTests.cs [hunk 1]
```diff
@@ -9,7 +9,9 @@ namespace Ryujinx.Tests.Memory
 {
     public class MultiRegionTrackingTests
     {
+#pragma warning disable IDE0051 // Remove unused private member
         private const int RndCnt = 3;
```
Comment (AcK77) [inline src/Ryujinx.Tests.Memory/MultiRegionTrackingTests.cs:13]: If this const is unused, maybe we should remove it ?

File: src/Ryujinx.Tests.Memory/TrackingTests.cs [hunk 2]
```diff
@@ -102,14 +102,14 @@ public void OverlappingRegions()
...
-            Action registerReadAction = () =>
+            void registerReadAction()
             {
                 readTrackingTriggeredAll = null;
                 allHandle.RegisterAction((address, size) =>
                 {
                     readTrackingTriggeredAll = (address, size);
                 });
-            };
+            }
```
Comment (AcK77) [inline src/Ryujinx.Tests.Memory/TrackingTests.cs:112]:
```suggestion
            }

```

File: src/Ryujinx.Tests.Memory/TrackingTests.cs [hunk 3]
```diff
@@ -102,14 +102,14 @@ public void OverlappingRegions()
             allHandle.Reprotect();
 
             (ulong address, ulong size)? readTrackingTriggeredAll = null;
-            Action registerReadAction = () =>
+            void registerReadAction()
```
Comment (AcK77) [inline src/Ryujinx.Tests.Memory/TrackingTests.cs:105]:
```suggestion

            void registerReadAction()
```

**Review by AcK77 — APPROVED** [review]
Lgtm after comments fixed.

**Review by marysaka — APPROVED** [review]

**Pull request merged**